config/internal_router/gen_keys.sh

```diff
@@ -5,6 +5,7 @@ INTERNAL_NETWORK="192.168.0.0/20"
 TEAM_IP_PREFIX="192.168.1."
 ROUTER_ADDRESS="192.168.0.2/20"
 ENGINE_ADDRESS="192.168.1.0/32"
+MOLOCH_ADDRESS="192.168.0.3/20"
```

Comment:
This should be `/32`, otherwise a route for `192.168.0.0/20 via 192.168.0.3` will be added which conflicts with the default router

Reply:
Ok, copied that from the router and wasnt sure with the ip